### PR TITLE
Update plot module

### PR DESCRIPTION
**Pull request by sturkmen72**
improving #1367

### CODE REVIEW

**Comment by sovrasov** [comment]
:+1:

File: modules/plot/samples/plot_demo.cpp [hunk 1]
```diff
@@ -6,10 +6,10 @@ using namespace cv;
...
 int main()
 {
-    Mat data_x(1, 50, CV_64F);
-    Mat data_y(1, 50, CV_64F);
+    Mat data_x(1, 51, CV_64F);
+    Mat data_y(1, 51, CV_64F);
 
-    for (int i = 0; i < 50; i++)
+    for (int i = 0; i < 51; i++)
     {
```
Comment (LaurentBerger) [inline modules/plot/samples/plot_demo.cpp]: may be  for (int i = 0; i <data_x.cols; i++) is better

Comment (sturkmen72) [reply modules/plot/samples/plot_demo.cpp]: @LaurentBerger you are right. let me change it asap.

**Comment by sturkmen72** [comment]
@sovrasov please review updated plot_demo.cpp

**Comment by sovrasov** [comment]
:+1:

**Pull request merged**